GoogleForCreators/web-stories-wp
# Fix media double-click handler

## Context
The media double-click handler was never used due to the click event not reaching there.
This PR moves the click handler together with the general `mouseDown` handler.


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Add media (try with video and image, background and regular element)
2. Try entering edit mode via double-click, verify it works.


### QA






This PR can be tested by following these steps:

1. Add media (try with video and image, background and regular element)
2. Try entering edit mode via double-click, verify it works.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6163
